test/SILOptimizer/access_enforcement_selection.swift

```diff
@@ -21,8 +26,8 @@ public func captureStack() -> Int {
   return x
 }
 // CHECK-LABEL: Access Enforcement Selection in _T028access_enforcement_selection12captureStackSiyF
-// Static access for `return x`.
-// CHECK: Static Access: %{{.*}} = begin_access [read] [static] %{{.*}} : $*Int
+// Dynamic access for `return x`.
+// CHECK: Dynamic Access: %{{.*}} = begin_access [read] [dynamic] %{{.*}} : $*Int
```

Comment:
It would help me to make a distinction between the requirements of the model vs. the current implementation. One way to do that is to add a comment on each test that points out how it is conservative. I think this is one of those places.